src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1543820426.inc.php

```diff
@@ -0,0 +1,56 @@
+<h1>Build #1543820426</h1>
+<h2>Date: 2018-12-03</h2>
+<div class="changelog">
+    - #135: Add cronjobs enabled flag
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'name' => 'cronjobs_enabled',
+      'translation' => 'Cronjobs erlaubt',
```

Comment:
"erlaubt" --> "aktiv" (I think you tried to avoid confusion with the "active" term for single cronjobs, but "erlaubt" doesn't really fit; I'm open for other suggestions though)

Reply:
Changed (back) to "aktiv".

Other suggestion: "Cronjob-Ausführung erlaubt"

Reply:
I'm good with the current text.